userinterface/Views/Profile/AccelerationFormulaSettingsView.axaml.cs

```diff
@@ -0,0 +1,231 @@
+﻿using Avalonia.Controls;
+using Avalonia.Data;
+using Avalonia.Interactivity;
+using Avalonia.Layout;
+using System;
+using userinterface.ViewModels.Profile;
+using userinterface.ViewModels.Controls;
+using userinterface.Views.Controls;
+using BEData = userspace_backend.Data.Profiles.Accel.FormulaAccel;
+
+namespace userinterface.Views.Profile;
+
+public partial class AccelerationFormulaSettingsView : UserControl
+{
+    private const int SynchronousFormulaIndex = 0;
```

Comment:
These indices assume a certain order of BEData.AccelerationFormulaType, per code in GetCurrentFormulaTypeIndex. However someone at some point may change BEData.AccelerationFormulaType and not realize they need to change here as well.
These indices should either be replaced with BEData.AccelerationFormulaType outright, or have their own enum with a mapping from BEData.AccelerationFormulaType (if for example index order change is desired) to formalize the relationship.